site/content/en/docs/manual/advanced/automatic-annotation.md

```diff
@@ -2,54 +2,106 @@
+- [Labels matching](#labels-matching)
+- [Models](#models)
+- [Adding models from Hugging Face and Roboflow](#adding-models-from-hugging-face-and-roboflow)
+
+## Running Automatic annotation
+
+To start automatic annotation, do the following:
+
+1. On the top menu, click **Tasks**.
+2. Find the task you want to annotate and click **Action** > **Automatic Annotation**.
 
    ![](/images/image119_detrac.jpg)
 
-1. In the dialog window select a model you need. DL models are created for specific labels, e.g.
-   the Crossroad model was taught using footage from cameras located above the highway and it is best to
-   use this model for the tasks with similar camera angles.
-   If it's necessary select the `Clean old annotations` checkbox.
-   Adjust the labels so that the task labels will correspond to the labels of the DL model.
-   For example, let’s consider a task where you have to annotate labels “car” and “person”.
-   You should connect the “person” label from the model to the “person” label in the task.
-   As for the “car” label, you should choose the most fitting label available in the model - the “vehicle” label.
-   If the chosen model supports automatic attributes detection
-   (like facial expressions, for example: ``serverless/openvino/omz/intel/face-detection-0205``),
-   you can also map attributes between the DL model and your task.
-   The task requires to annotate cars only and choosing the “vehicle” label implies annotation of all vehicles,
-   in this case using auto annotation will help you complete the task faster.
-   Click `Submit` to begin the automatic annotation process.
+3. In the Automatic annotation dialog, from the drop-down list, select [model](#models).
+4. [Match labels](#labels-matching) of the model and the task.
+5. (Optional) In case you need the model to return masks as polygons, switch toggle **Return masks as polygons**.
+6. (Optional) In case you need to remove all previous annotations, switch toggle **Clean old annotations**.
 
    ![](/images/image120.jpg)
 
-1. At runtime - you can see the percentage of completion.
-   You can cancel the automatic annotation by clicking on the `Cancel`button.
+7. Click **Annotate**.
+
+CVAT will show the progress of annotation on the progress bar.
+
+![Progress bar](/images/image121_detrac.jpg)
+
+You can stop the automatic at any moment by clicking **Cancel**.
+
+## Labels matching
+
+Each model is trained on a dataset and supports only the dataset's labels.
+
+For example:
+
+- DL model has the label `car`.
+- Your task (or project) has the label `vehicle`.
+
+To annotate, you need to match these two labels to give
+DL model a hint that, in this case, `car` = `vehicle`.
+
+If you have a label that is not on the list
+of DL labels, you will not be able to
+match them.
+
+For this reason, supported DL models are suitable only
+for certain labels.
+
+<br>To check the list of labels for each model, see information about [Models](#models).
```

Comment:
* Why is there a `<br>` here?
* I looked at the information, but there's no mention of labels.

Reply:
mention of labels are in the papers. we do not add them straight to our docs, because they are not a part of CVAT. it is the same approach for ai tools, you can check it there. I will add explanation that you need to check the models papers.